src/server/pps/server/api_server.go

```diff
@@ -64,6 +66,8 @@ const (
 	// DefaultDatumTries is the default number of times a datum will be tried
 	// before we give up and consider the job failed.
 	DefaultDatumTries = 3
+
+	maxLogMessages = 5000
```

Comment:
could you add a comment describing this?

Reply:
Added a comment explaining this value, also moved it into the lokiutil package which is where it's now used.